test/dvotingdela.go

```diff
@@ -0,0 +1,510 @@
+package integration
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/x509"
+	"io"
+	"math/rand"
```

Comment:
Didn't check, but we should be sure to set a seed so that tests are reproducible